ssh/kex.go

```diff
@@ -33,14 +35,16 @@ const (
 	kexAlgoDHGEXSHA256 = "diffie-hellman-group-exchange-sha256"
 )
 
+// This is aligned with the preferredKeyAlgos from golang/crypto but includes kexAlgoDHGEXSHA256.
+
 var PreferredKexAlgos = []string{
 	kexAlgoCurve25519SHA256,
 	kexAlgoECDH256, kexAlgoECDH384, kexAlgoECDH521,
 	kexAlgoDH14SHA1,
 	kexAlgoDHGEXSHA256,
 }
 
-// SetPreferredKeyAlgos sets default key algorithms on ClientConfig
+// SetPreferredKeyAlgos sets the PreferredKexAlgos algorithms on a given ClientConfig
```

Comment:
```suggestion
// SetPreferredKeyAlgos sets the PreferredKexAlgos on a given ClientConfig.
```

I think the `Algos` in `PreferredKexAlgos` is already descriptive for "algorithms", and doc block comments should end with a dot.